srcpkgs/pandoc/template

```diff
@@ -1,31 +1,32 @@
 # Template file for 'pandoc'
 pkgname=pandoc
 # Keep in sync with http://www.stackage.org/lts
-version=2.2.1
+version=2.5
 revision=1
-_citeproc_version=0.14.3.1
+_citeproc_version=0.15.0.1
+create_wrksrc=yes
 build_style=haskell-stack
-stackage="lts-12.0"
 make_build_args="
  --flag pandoc:embed_data_files --flag pandoc-citeproc:embed_data_files"
-nocross=yes
-create_wrksrc=yes
-hostmakedepends="unzip"
+stackage="lts-13.1"
+hostmakedepends="unzip wget"
 makedepends="zlib-devel"
 short_desc="Universal converter between markup formats (with citeproc support)"
 maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2"
 homepage="http://johnmacfarlane.net/pandoc/"
 distfiles="http://hackage.haskell.org/package/${pkgname}-${version}/${pkgname}-${version}.tar.gz
  http://hackage.haskell.org/package/${pkgname}-citeproc-${_citeproc_version}/${pkgname}-citeproc-${_citeproc_version}.tar.gz"
-checksum="fe037f5fbb62fb27e7b1dbddfbd0aa45ea6e9fcdaff1f2203f7484c245b211b7
- 42c0b2c8365441bf884daa6202e6ed01b42181cf255406c88b3b31cd27cb467a"
+checksum="d57dc5db78a0a304de70436fe59a7599ab0c6d0fb2ab6704eeae498a4536222e
+ 29db5f2aad3225859727271855461724574f3695ab3856ceac33b24a55ae92f8"
+nocross=yes
 nopie_files="
  /usr/bin/pandoc
  /usr/bin/pandoc-citeproc
 "
 
 post_extract() {
+	wget https://hackage.haskell.org/package/pandoc-2.5/revision/2.cabal -O pandoc-2.5/pandoc.cabal
```

Comment:
This should be fetched via distfiles and then moved to it's needed path.

Reply:
Done